src/main/java/net/atos/zac/app/zaken/model/RESTZaak.java

```diff
@@ -71,6 +81,7 @@ public class RESTZaak {
 
     public String duurVerlenging;
 
+    @NotNull(message = "Group is required")
```

Comment:
Is niet waar? Groep is volgens de backend code optioneel:

``` if (restZaak.groep != null) {```

Reply:
anders gezegd: als ie wel verplicht is dan moeten we de backend code aanpassen denk ik. ik zou deze laten zitten hier